recipes/spleeter-gpu/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "spleeter-gpu" %}
+{% set version = "1.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 49e776f26c561357eb9608a459030ab46cfe11edf21fb17470630a0d3777d453
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: noarch
+
+requirements:
+  host:
+    - python >=3.6,<3.8
+    - pip
+  run:
+    - python >=3.6,<3.8
+    - requests
+    - setuptools
+    - importlib_resources  # [py<37]
+    - tensorflow-gpu ==1.14.0
+    - pandas ==0.25.1
+    - ffmpeg-python
+    - norbert
+
+test:
+  imports:
+    - spleeter
+
+about:
+  home: https://github.com/deezer/spleeter
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'The Deezer source separation library with pretrained models based on tensorflow.'
+  doc_url: https://github.com/deezer/spleeter/wiki
+  dev_url: https://github.com/deezer/spleeter
+
+extra:
+  recipe-maintainers:
+    - Faylixe
+    - romi1502
```

Comment:
@romi1502 Can you comment if you're happy to be listed as a maintainer of this?

Reply:
Ok for me :)